src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -95,8 +95,8 @@ const EntityViewerForGene = () => {
       mainContent={<GeneView />}
       topbarContent={
         <EntityViewerTopbar
-          genomeId={genomeId as string}
-          entityId={entityId as string}
+          genomeId={activeGenomeId}
+          entityId={activeEntityId}
```

Comment:
Can we use `useEntityViewerIds` in EntityViewerTopbar rather than passing the props down?

Reply:
✅